models/bases/procedures.sql

```diff
@@ -1,8 +1,8 @@
 select
     p.id,
-    p.date::date as date,
-    p.start_time::time as start_time,
-    p.end_time::time as end_time,
+    p.date::timestamp as datetime,
+    p.start_time::timestamp as start_time,
+    p.end_time::timestamp as end_time,
```

Comment:
Based on the front end, we would only use the date from the date field and the time from the start_time and end_time.
```suggestion
    p.date::date,
    p.start_time::time,
    p.end_time::time,
```